curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md

```diff
@@ -65,12 +65,12 @@ tests:
 <div id='js-seed'>
 
 ```js
-/* Alter code below this line */
+// Only change code below this line
 
-/* Alter code above this line */
+// Only change code above this line
 
 const carrot = new Vegetable('carrot');
-console.log(carrot.name); // => should be 'carrot'
+console.log(carrot.name); // Should display 'carrot'
```

Comment:
It should look for the exact comment `Should display 'carrot'`

Reply:
The reason is, we do not necessarily know where the Chinese equivalent would place `carrot`.  It might be there would need to be Chinese characters before and after it to make sense in Chinese.